src/EnergyPlus/SingleDuct.cc

```diff
@@ -391,11 +390,13 @@ void GetSysInput(EnergyPlusData &state)
         } else if (UtilityRoutines::SameString(state.dataSingleDuct->sd_airterminal(state.dataSingleDuct->SysNumGSI).ReheatComp,
```

Comment:
`getEnumerationValue`.

Reply:
Will tackle once enum work for DataHVACGlobals.cc are done. These strings are already part of arrays in that file, so I can hopefully reference them and avoid some duplicates.